go/install/sources/prerelease.go

```diff
@@ -0,0 +1,8 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+// +build prerelease
```

Comment:
yeah prereleases can happen more often than nightly so calling it prerelease...  but yeah these are the same builds as nightly